### PR TITLE
feat: TokenCard & FeatureCard

### PR DESCRIPTION
**Pull request by diegonzs-roll**
## What's done
- Add TokenCard component
- Add FeatureCard component
## Tasks

- [x] Have you written tests for new code (if applicable)?
- [x] Have you tested the changes on all the platforms?
  - [x] iOS
  - [ ] Android
  - [x] Web
- [x] Have you added stories to demonstrate the new functionality (if there is any)?

## Demo (screenshots, recordings, etc.)
https://www.loom.com/share/4729f913f8cf4915a4b50db00dedc01f

### CODE REVIEW

File: packages/design-system/src/hooks/utils.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useClipboard } from 'native-base'
```
Comment (teimurjan) [inline packages/design-system/src/hooks/utils.ts:1]: Let's rename the file to `clipboard.ts` to keep here all the clipboard-related hooks & avoid a generic name like `utils`.

File: packages/design-system/src/hooks/utils.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { useCallback } from 'react'
+import { Toast } from '../atoms'
+
+export const useCopyOnClipboard = () => {
```
Comment (teimurjan) [inline packages/design-system/src/hooks/utils.ts:5]: Let's rename the hook to be more explicit.

```suggestion
export const useClipboardWithToast = () => {
```

File: packages/design-system/src/molecules/tokenCard/tokenCard.test.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        name={TOKEN_NAME}
+        symbol={TOKEN_SYMBOL}
+        chainId={CHAIN_ID_MAIN_NET}
+      />,
+      {
+        wrapper: TryrollTestProvider,
+      },
+    )
+  })
+  it('Renders short address', async () => {
```
Comment (teimurjan) [inline packages/design-system/src/molecules/tokenCard/tokenCard.test.tsx:37]: Can you use lowercase letters for `it`, please? We have to be consistent within the codebase

**Review by teimurjan — COMMENTED** [review]
Why is blurred text padded on left?

Screenshot 2023-04-03 at 19 48 08

**Comment by diegonzs-roll** [comment]
> Why is blurred text padded on left?
> 
> Screenshot 2023-04-03 at 19 48 08

The SVG already has that internal padding I already ask @andreider04 for the SVG :)

**Pull request merged**